config/README.md

```diff
@@ -1,135 +1,169 @@
-# Multi-region cockroachdb setup
+# Deploying a DSS
 
-## Prerequisites
 
-Download & install:
 
-*   helm
-*   kubectl
-*   docker
-*   cockroachdb
-*   Google Cloud SDK (if deploying on GCP)
-*   [Optional] Golang. Recommended to understand go, and the go toolchain.
+## Glossary
 
+*   DSS cluster - an entire synchronized DSS, typically operated by multiple
```

Comment:
As below, let's keep our terminology consistent with the standard.  So, "DSS Region - The geographic scope supported by a set of Discovery and Synchronization Service instances." (definition from standard) and "DSS instance - a single deployed implementation of the DSS interface which synchronizes with other DSS instances in its DSS Region." (not replica because there can be different implementations)

Reply:
I'm going to push back on the term instance, cause when I hear instance I think of the entire cluster, whereas somebody else might think of a node -- ie: it's much too ambiguous

I think DSS Node is a good term, as well as cluster. DSS region to me also means the serviced region, so would like to pick something more specific there as well.

Also this terminology more closely aligns with that of CockroachDB, so a +1 for cluster and Node.

Reply:
FYI https://www.cockroachlabs.com/docs/stable/architecture/overview.html#glossary

Reply:
done